src/WingetCreateCore/Common/GitHub.cs

```diff
@@ -361,7 +361,15 @@ private async Task<PullRequest> SubmitPRAsync(string packageId, string version,
                 // Remove a previous manifest
                 if (shouldReplace)
                 {
-                    await this.DeletePackageManifest(repo.Id, packageId, replaceVersion, newBranchName);
+                    try
+                    {
+                        await this.DeletePackageManifest(repo.Id, packageId, replaceVersion, newBranchName);
+                    }
+                    catch (NotFoundException)
+                    {
+                        // If the path to the manifest folder being replaced does not exist, do nothing.
+                        // This can happen with submit command when user submits a new package and path to its version directory does not exist yet.
+                    }

```

Comment:
We should be explicitly checking whether the folder being replaced exists or not rather than using a try-catch exception to handle that scenario.

Reply:
Made the flow similar to update command where we validate replace arguments prior to going into GitHub submit flow.